api/services/EmailTemplates/forgottenUsername.ejs

```diff
@@ -0,0 +1,3 @@
+<p>
+  Your username for your account at <a href="http://<%= siteUrl %>"><%= siteUrl %></a> is <%= recipientName %>.
```

Comment:
Damnit, why the hell do we allow mutliple users to have the same email address? Well, I guess this wouldn't work...